docs/glossary.md

```diff
@@ -857,4 +857,15 @@ Vale
     [Vale](https://vale.sh/) is an open-source, command-line tool that helps maintain a consistent and on-brand voice in documentation.
     Plone Documentation uses it to check spelling, English grammar and syntax, and style guides.
 
+schema enhancer
+    A schema enhancer uses the `schemaEnhancer` function, which can modify the schema used by the `InlineForm` component.
+    Any registered extension plugin can provide a `schemaEnhancer` function.
+    This function receives an object with `formData`, which is the block data; `schema`, which is the original schema to modify; and the injected `intl`, which aids with internationalization.
+
+variation
+    A variation is a common development pattern that provides versions of a given block.
+    You can use a variation to enhance the default behavior of a block without shadowing its stock components.
+    These enhancements can either be at the settings level via schema enhancers or, if the code of your block allows it, use alternative renderers showing the enhanced fields, modifying the block behavior or its look and feel.
+    Editors can select a variation on demand.
+
```

Comment:
I would make this last definition simpler.

The standard pattern for variations is to offer "alternative views" for the same data. Example: A variation shows  news items with "title + link". Another variation shows "title + description + link", another variations shows "title + image + link".

As an advanced pattern we can say that a variation can enhance the block adding additional fields to the block. Example: 1 variation shows news items with "title + link". Another variation adds an additional boolean field to the block to decide whether to show the link. So in the end the editor can select between "title + link" and "title + link (if the boolean is set to true)".

Reply:
> "title + link" and "title + link (if the boolean is set to true)".

Aren't these the same?